gcc-4.8/meta.yaml

```diff
@@ -17,14 +17,14 @@ requirements:
     - gmp >=4.3.2
     - mpfr >=2.4.2
     - mpc >=0.8.1
-    - isl
+    - isl >=0.10,<=0.14
     - cloog 0.18.0
     # Do not make gcc a build dependency (you will need to add it to the PATH manually)
   run:
-    - gmp >=4.2
-    - mpfr >=2.4.0
-    - mpc >=0.8.0
```

Comment:
@ccordoba12 How much sense does it make using older run dependencies than build dependencies?